docs/modules/ROOT/nav.adoc

```diff
@@ -283,6 +282,7 @@
 
 .Appendices
 * xref:release-notes/release-notes-history.adoc[History of Release Notes]
+** xref:release-notes/release-notes-23-1.adoc[Payara Enterprise 5.24.0 Release Notes]
```

Comment:
You can remove this line.

First, the link is wrong (points to 5.23.1). Second, the 5.24 release notes is already linked in the main area. Duplicate links don't work well in Antora.